docs/sources/setup-grafana/configure-security/configure-authentication/github/index.md

```diff
@@ -97,6 +98,9 @@ If no valid role is found, the user is assigned the role specified by [the `auto
 You can disable this default role assignment by setting `role_attribute_strict = true`.
 This setting denies user access if no role or an invalid role is returned.
 
+The `org_roles_attribute_path` configuration option can be used to add roles to other Orgs. See [Org roles mapping examples]
```

Comment:
You can use the `org_roles_attribute_path` configuration option to add roles to other orgs. For more information, refer to [Org roles mapping examples]({{< relref "#org-roles-mapping-examples" >}}).